### PR TITLE
LRQA-63963 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-63963

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:relevant

**Comment by nkkwon** [comment]
ci:test:sf

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-test/src/testFunctional/macros/AppBuilderWorkflow.macro [hunk 1]
```diff
@@ -277,6 +277,10 @@ definition {
 			locator1 = "IFrame#MODAL_ANY");
 	}
 
+	macro collapseActivity {
```
Comment (rodrigocunhaa) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-test/src/testFunctional/macros/AppBuilderWorkflow.macro:280]: Please, change this macro name to `collapseOrExpandActivity`

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-test/src/testFunctional/tests/AppBuilderWorkflow.testcase [hunk 2]
```diff
@@ -253,6 +253,77 @@ definition {
...
+		AppBuilderWorkflow.selectStepSingleFormView(
+			formViewMode = "Editable",
+			formViewName = "My Form View",
+			step = "1");
+
+		AppBuilderWorkflow.addNewStep();
+
+		AppBuilderWorkflow.selectStepAssignee(
+			assignee = "Administrator",
+			step = "2");
```
Comment (rodrigocunhaa) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-test/src/testFunctional/tests/AppBuilderWorkflow.testcase:298]: There is no need for these steps, you can remove it.

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @nkkwon ,

Can you please check the changes that I mentioned?

Besides those changes, there is also one scenario that you missed:

- Assign the entry to yourself with some comments and assert that the comment is present on the Activity section.

Also, please fix the conflicts. Thanks!

**Comment by nkkwon** [comment]
See https://github.com/rodrigocunhaa/liferay-portal/pull/313

**Pull request closed without merging**